src/lib/schematics/tree/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,22 @@
+<mat-tree [dataSource]="dataSource" [treeControl]="treeControl">
+  <mat-tree-node *matTreeNodeDef="let node" matTreeNodeToggle matTreeNodePadding>
+    <button mat-icon-button disabled></button>
```

Comment:
Could we omit this (empty/nonfunctional) button completely and just capture the spacing with a different element?

Reply:
I just copied what the docs examples did. Open to other ideas though. Just let me know.

Reply:
cc @tinayuangao there's no reason this needs to be a button, right? It's just to take up space?